### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_Create a Scala Project_Screen Reader: On changing the Java SE under JRE, a message is bein displayed. Narrator and NVDA are not reading it

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#4297]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-Benchmark; #MAS4.1.3; #ScreenReader; #JavatoolingforIntelliJ/Eclipse; #C+E-H; #Benchmark; #A11ySev1; #Narrator; #NVDA 

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2020-03 (4.15.0)
Build id: 20200313-1211

**Solution Installed:** The Azure Toolkit for Eclipse 3.220, Scala IDE 4.7.x

**OS Details:**
OS: Windows (Version 1903, Build: 18362.175)

**Screen Reader:** Narrator, NVDA

**MAS Violated:** MAS4.1.3

**Repro Steps:**
1.Start Screen Reader. Open Java EE - Eclipse application 
2.Install "Azure Toolkit for Eclipse" and "Scala Plug-ins" from Eclipse Marketplace
3.Tab till "File > New > Project" in the navigation bar and press enter
4.Tab till "HDInsight Project > Spark on HDInsight (Scala)" and select it.
5.Tab till "Next" button and press enter
6.Under JRE, tab till the combo-box against "Use an execution environment JRE" radio button 
7.Verify if the screen reader is reading the alert message that is displayed on changing the selection of the combo-box

**Actual Result:**
Narrator and NVDA are not reading the status message that is displayed on changing the selection of the combo-box

**Expected Result:**
Narrator and NVDA should read the status message that is displayed on changing the selection of the combo-box

**User impact:**
Screen Reader users will not be notified the issue related to that particular selection if the message is not read

**MAS Reference:**
[MAS 4.1.3](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7BC154FBE5-702F-42FB-B2BC-87FBB3A23D91%7D&file=MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx&action=default&mobileredirect=true)

![Bug_4297 _Java Tooling_Create a Scala Project_Status Message_NVDA](https://user-images.githubusercontent.com/41108985/79102264-03bd8e00-7d88-11ea-9539-a32b14734b97.png)

[Bug_4297 _Java Tooling_Create a Scala Project_Status Message_Narrator.pptx](https://github.com/microsoft/azure-tools-for-java/files/4469411/Bug_4297._Java.Tooling_Create.a.Scala.Project_Status.Message_Narrator.pptx)

**Comment by jingyanjingyan** [comment]
This wizard is extends from original Eclipse wizard. You can find the same dialog in New Project > Java > Java Project. So it's out of the scope of our toolkit. Please find the screenshot of Eclipse without our toolkit.
![access4](https://user-images.githubusercontent.com/36756024/81459253-8c85e900-91d1-11ea-91ba-7480471db512.png)

**Comment by jingyanjingyan** [comment]
+ Importance of experience: Minimal. Screen Reader users will not be notified the issue related to that particular selection if the message is not read
+ Quality of experience: Annoying. The narrator could read the value changed and user could perform create process.
+ Alternate solutions: This issue will not break users experience, no need for alternate solutions
+ Evidence: All evidence is provided in this issue.
+ [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

External issues tracked in: https://bugs.eclipse.org/bugs/show_bug.cgi?id=562984

**Comment by Alexandria-R** [comment]
Known eclipse issue, approved to close

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749648
